stage1/init/common/pod.go

```diff
@@ -348,6 +380,18 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
+	if flavor == "kvm" {
+		capabilities := kvm.GetAppCapabilities(app)
+		capabilities = append(capabilities, stdCapabilities...)
```

Comment:
So are you just trying to preserve parity with the fact that a process launched by a `systemd`-launched-by-`systemd-nspawn` will have different default capabilities to one launched by `systemd`-launched-by-lkvm?

If so, wouldn't the better approach be to change the systemd config for the lkvm variant? e.g. https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#CapabilityBoundingSet=